src/services/auth-http.ts

```diff
@@ -15,3 +18,21 @@ export const cloudSettings: CloudSettings = {
     'app_id': 'APP_ID'
   }
 };
+
+export function handleError(error: Response | any, caught: any) {
```

Comment:
It was called `auth-http` because it originally only contained stuff relative to the set up of the auth-http module. But you are right that it makes sense to rename it now. I renamed it to `auth-http-helpers`, because it is still relative to auth-http.